ResearchKit/Common/ORKOrderedTask.m

```diff
@@ -107,6 +107,15 @@ - (NSUInteger)hash {
     return [_identifier hash] ^ [_steps hash];
 }
 
+- (ORKStep *)removeStepAtIndex:(NSUInteger)index {
+    NSParameterAssert(index < _steps.count);
+    NSMutableArray *mutableSteps = [_steps mutableCopy];
+    ORKStep *ret = [mutableSteps objectAtIndex:index];
```

Comment:
RK doesn't normally use the `ret` naming convention. Perhaps name it `removedStep`?